client/app/src/accounts/view/createSecondPassphrase.html

```diff
@@ -30,8 +31,11 @@
           <input ng-if="!createSecondPassphraseDialog.data.showRepassphrase" ng-model="createSecondPassphraseDialog.data.secondPassphrase" type="text" readonly="readonly" md-autofocus/>
           <input ng-if="createSecondPassphraseDialog.data.showRepassphrase" ng-model="createSecondPassphraseDialog.data.secondPassphrase" type="text" ng-required="true" md-autofocus/>
         </md-input-container>
-
-        <div ng-if="createSecondPassphraseDialog.data.showWrongRepassphrase" translate>
+        <div ng-if="createSecondPassphraseDialog.data.showRepassphrase">
+          <translate>Fee</translate>: {{ul.network.symbol}}5
```

Comment:
What's going on with the config service? It'd be nice if I could just use something like {{app.config.secondPassphraseFee}} instead of "5".